src/blob/handlers/delete.handler.ts

```diff
@@ -0,0 +1,26 @@
+import type { FastifyReply, FastifyInstance } from "fastify";
+import { BlobService } from "../services/blob.service.js";
+import type { BlobRequest } from "../types.js";
+
+export async function deleteBlobHandler(
+  request: BlobRequest,
+  reply: FastifyReply,
+  fastify: FastifyInstance
+): Promise<FastifyReply> {
+  try {
+    const deleted = await BlobService.deleteBlob(request.params.id);
+
+    if (!deleted) {
+      return reply.code(200).send();
+    }
+
+    return reply.code(204).send();
```

Comment:
IMO it's more confusing to have 2 different status codes for this action. but hey, it's your API! 😄